docker-compose.yml

```diff
@@ -138,6 +138,12 @@ services:
     depends_on:
       kafka3:
         condition: service_healthy
+    logging:
```

Comment:
Thanks, Zahra; I wasn't aware of this functionality in docker-compose. Please add this to all other services